test/specs/controller.doughnut.tests.js

```diff
@@ -325,6 +325,83 @@ describe('Chart.controllers.doughnut', function() {
 		expect(controller.innerRadius).toBe(126);
 	});
 
+	it('should create arcs with widths based on the weight value of the dataset', function() {
```

Comment:
I think an image based test would have been better here, @nagix what do you think?

Reply:
@nagix @simonbrunel Do you want to add an image based test?  I can look into it if need be,

Reply:
I would replace these pixel based tests by image based tests since it's more robust and (IMO) easier to update. We are actually [writing some docs](https://github.com/chartjs/Chart.js/pull/6073) about this type of tests, you could give a try :)

Reply:
@Vincent-Ip do you think we need to keep this test in `controller.doughnut.tests.js` since it's exactly the same as the fixture ones? Except if it tests something special, I think I would just remove it to avoid maintaining both later.

Reply:
@simonbrunel Well, the pixel based tests do overlap with the image test, but the pixel tests are much more explicit, in terms of testing those newly added private methods and their resulting behavior.  Since these methods are private, I guess testing could be thought of as overkill, since no one is relying on their behavior, and the image based tests cover the general overall behavior.

I do want to say that there are similar existing methods that are public (and this code was originally written emulating existing public methods), so I could see that if any of these methods become part of a public api then it may call for the re-introduction of pixel based test coverage code.

But since these methods are currently private, we can let whatever future PR re-introduce pixel based tests, if needed.  **I will remove the pixel based tests**.  Apologies to all for the constant little updates and making you all have to re-approve this PR, but I guess that's just how the sausage is made.